<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(40, 40, 40);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="InsertCardLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>431</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #002800;      /* Green border */
    border-radius: 5px;            /* Rounded corners */
    padding: 5px;                   /* Inner spacing */
    background-color: gray;      /* Optional: light background */
    color: #ffffff;    
font: bold 16px;    </string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Insert Card&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="TestButton">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>190</y>
          <width>141</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Login to account</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QPushButton" name="button1">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>170</y>
          <width>81</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">  background-color: gray; 
    border-radius: 40px;
    color: white;
    font: bold 16px;</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
       <widget class="QLabel" name="inputMenuLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>431</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #002800;      /* Green border */
    border-radius: 5px;            /* Rounded corners */
    padding: 5px;                   /* Inner spacing */
    background-color: gray;      /* Optional: light background */
    color: #ffffff;    
font: bold 16px;    </string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Log In&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>170</y>
          <width>81</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">  background-color: gray; 
    border-radius: 40px;
    color: white;
    font: bold 16px;</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button3">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>170</y>
          <width>81</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">  background-color: gray; 
    border-radius: 40px;
    color: white;
    font: bold 16px;</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pinline">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>120</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #002800;      /* Green border */
    border-radius: 5px;            /* Rounded corners */
    padding: 5px;                   /* Inner spacing */
    background-color: gray;      /* Optional: light background */
    color: white;        
font: bold 16px;</string>
        </property>
       </widget>
       <widget class="QLabel" name="inputMenuLabel_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>421</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: bold 16px;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Enter PIN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button4">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>270</y>
          <width>81</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">  background-color: gray; 
    border-radius: 40px;
    color: white;
    font: bold 16px;</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button5">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>270</y>
          <width>81</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">  background-color: gray; 
    border-radius: 40px;
    color: white;
    font: bold 16px;</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button6">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>270</y>
          <width>81</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">  background-color: gray; 
    border-radius: 40px;
    color: white;
    font: bold 16px;</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button7">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>370</y>
          <width>81</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">  background-color: gray; 
    border-radius: 40px;
    color: white;
    font: bold 16px;</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button8">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>370</y>
          <width>81</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">  background-color: gray; 
    border-radius: 40px;
    color: white;
    font: bold 16px;</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button9">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>370</y>
          <width>81</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">  background-color: gray; 
    border-radius: 40px;
    color: white;
    font: bold 16px;</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button0">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>470</y>
          <width>81</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">  background-color: gray; 
    border-radius: 40px;
    color: white;
    font: bold 16px;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QPushButton" name="inputBackButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>3</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #002800;      /* Green border */
    border-radius: 5px;            /* Rounded corners */
    padding: 5px;                   /* Inner spacing */
    background-color: gray;      /* Optional: light background */
    color: #ffffff;        </string>
        </property>
        <property name="text">
         <string>&lt;BACK</string>
        </property>
       </widget>
       <widget class="QPushButton" name="DELbutton">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>480</y>
          <width>80</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 4px solid #610602;     
 font: bold 16px;
    border-radius: 5px;           
    padding: 5px;                   
    background-color: #d96762;      
    color: #2d0000;        </string>
        </property>
        <property name="text">
         <string>&lt;DEL</string>
        </property>
       </widget>
       <widget class="QPushButton" name="OKbutton">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>480</y>
          <width>80</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 4px solid #03822d;      /* Green border */
font: bold 16px;
    border-radius: 5px;            /* Rounded corners */
    padding: 5px;                   /* Inner spacing */
    background-color: #5dba7d;      /* Optional: light background */
    color: #002800;        </string>
        </property>
        <property name="text">
         <string>OK&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QLabel" name="menulabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>431</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #002800;      /* Green border */
    border-radius: 5px;            /* Rounded corners */
    padding: 5px;                   /* Inner spacing */
    background-color: gray;      /* Optional: light background */
    color: #ffffff;    
font: bold 16px;    </string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;MENU&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deposit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>370</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: bold 16px;
    border-radius: 10px;            /* Rounded corners */
    padding: 5px;                   /* Inner spacing */
    background-color: gray;      /* Optional: light background */
    color: white;     </string>
        </property>
        <property name="text">
         <string>Deposit</string>
        </property>
       </widget>
       <widget class="QPushButton" name="transactions">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>240</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: bold 16px;
    border-radius: 10px;            /* Rounded corners */
    padding: 5px;                   /* Inner spacing */
    background-color: gray;      /* Optional: light background */
    color: white;        </string>
        </property>
        <property name="text">
         <string>Transactions</string>
        </property>
       </widget>
       <widget class="QPushButton" name="withdraw">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>370</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: bold 16px;
    border-radius: 10px;            /* Rounded corners */
    padding: 5px;                   /* Inner spacing */
    background-color: gray;      /* Optional: light background */
  color: white;          </string>
        </property>
        <property name="text">
         <string>Withdraw</string>
        </property>
       </widget>
       <widget class="QPushButton" name="transfer">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>240</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: bold 16px;
    border-radius: 10px;            /* Rounded corners */
    padding: 5px;                   /* Inner spacing */
    background-color: gray;      /* Optional: light background */
      color: white;         </string>
        </property>
        <property name="text">
         <string>Transfer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="logout">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>510</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: bold 16px;
    border-radius: 10px;            /* Rounded corners */
    padding: 5px;                   /* Inner spacing */
    background-color: gray;      /* Optional: light background */
     color: white;           </string>
        </property>
        <property name="text">
         <string>Log Out</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="MenuBrowser">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>411</width>
          <height>91</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #002800;      /* Green border */
font: bold 16px;
    border-radius: 5px;            /* Rounded corners */
    padding: 5px;                   /* Inner spacing */
    background-color: gray;      /* Optional: light background */
    color: #ffffff;        </string>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:16px; font-weight:700; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Account:2&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Balance:2500&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Credit Limit:3000&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="CreditLimit">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>510</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: bold 16px;
    border-radius: 10px;            /* Rounded corners */
    padding: 5px;                   /* Inner spacing */
    background-color: gray;      /* Optional: light background */
     color: white;           </string>
        </property>
        <property name="text">
         <string>Edit Credit Limit</string>
        </property>
       </widget>
       <widget class="QPushButton" name="nextAccount">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>110</y>
          <width>31</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::GoNext"/>
        </property>
       </widget>
       <widget class="QPushButton" name="previousAccount">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>31</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::GoPrevious"/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QPushButton" name="transactionBack">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #002800;      /* Green border */
    border-radius: 5px;            /* Rounded corners */
    padding: 5px;                   /* Inner spacing */
    background-color: gray;      /* Optional: light background */
    color: #ffffff;        </string>
        </property>
        <property name="text">
         <string>&lt;BACK</string>
        </property>
       </widget>
       <widget class="QLabel" name="transactionslabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>431</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #002800;      /* Green border */
    border-radius: 5px;            /* Rounded corners */
    padding: 5px;                   /* Inner spacing */
    background-color: gray;      /* Optional: light background */
    color: #ffffff;    
font: bold 16px;    </string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;TRANSACTIONS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="RefreshButton">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #002800;      /* Green border */
    border-radius: 5px;            /* Rounded corners */
    padding: 5px;                   /* Inner spacing */
    background-color: gray;      /* Optional: light background */
    color: #ffffff;        </string>
        </property>
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
       <widget class="QComboBox" name="transactionFilter">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>90</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QScrollArea" name="scrollArea">
        <property name="geometry">
         <rect>
          <x>29</x>
          <y>149</y>
          <width>371</width>
          <height>401</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>369</width>
           <height>399</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
